swim/member.go

```diff
@@ -73,40 +85,49 @@ func shuffle(members []*Member) []*Member {
 	return newMembers
 }
 
-// nonLocalOverride returns wether a change should be applied to the member.
-// This function assumes that the address of the member and the change are
-// equal.
-func (m *Member) nonLocalOverride(change Change) bool {
-	// change is younger than current member
-	if change.Incarnation > m.Incarnation {
+// acceptGossip evaluates the rules of swim to accept the gossip as the new state
+// of the member.
+func acceptGossip(old *Member, gossip *Member) bool {
```

Comment:
I'm not sure if this name covers the intention: the gossip won't be accepted "as is" if it's overwriting the local member. This method just checks if the incoming gossip should be processed or could be ignored right?